src/resources/parser/glb-parser.js

```diff
@@ -0,0 +1,1466 @@
+                    var options = {};
+                    if (target.hasOwnProperty('POSITION')) {
+                        accessor = accessors[target.POSITION];
+                        options.deltaPositions = getAccessorData(accessor, bufferViews, buffers);
+                    }
+                    if (target.hasOwnProperty('NORMAL')) {
+                        accessor = accessors[target.NORMAL];
+                        options.deltaNormals = getAccessorData(accessor, bufferViews, buffers);
+                    }
+                    if (target.hasOwnProperty('TANGENT')) {
+                        accessor = accessors[target.TANGENT];
+                        options.deltaTangents = getAccessorData(accessor, bufferViews, buffers);
+                    }
+
+                    targets.push(new pc.MorphTarget(options));
+                });
+
+                mesh.morph = new pc.Morph(targets);
+            }
+
+            meshes.push(mesh);
+        });
+
+        return meshes;
+    };
+
+    var createMaterial = function (materialData, textures) {
+        var glossChunk = [
+            "#ifdef MAPFLOAT",
+            "uniform float material_shininess;",
+            "#endif",
+            "",
+            "#ifdef MAPTEXTURE",
+            "uniform sampler2D texture_glossMap;",
+            "#endif",
+            "",
+            "void getGlossiness() {",
+            "    dGlossiness = 1.0;",
+            "",
+            "#ifdef MAPFLOAT",
+            "    dGlossiness *= material_shininess;",
+            "#endif",
+            "",
+            "#ifdef MAPTEXTURE",
+            "    dGlossiness *= texture2D(texture_glossMap, $UV).$CH;",
+            "#endif",
+            "",
+            "#ifdef MAPVERTEX",
+            "    dGlossiness *= saturate(vVertexColor.$VC);",
+            "#endif",
+            "",
+            "    dGlossiness = 1.0 - dGlossiness;",
+            "",
+            "    dGlossiness += 0.0000001;",
+            "}"
+        ].join('\n');
+
+        var specularChunk = [
+            "#ifdef MAPCOLOR",
+            "uniform vec3 material_specular;",
+            "#endif",
+            "",
+            "#ifdef MAPTEXTURE",
+            "uniform sampler2D texture_specularMap;",
+            "#endif",
+            "",
+            "void getSpecularity() {",
+            "    dSpecularity = vec3(1.0);",
+            "",
+            "    #ifdef MAPCOLOR",
+            "        dSpecularity *= material_specular;",
+            "    #endif",
+            "",
+            "    #ifdef MAPTEXTURE",
+            "        vec3 srgb = texture2D(texture_specularMap, $UV).$CH;",
+            "        dSpecularity *= vec3(pow(srgb.r, 2.2), pow(srgb.g, 2.2), pow(srgb.b, 2.2));",
+            "    #endif",
+            "",
+            "    #ifdef MAPVERTEX",
+            "        dSpecularity *= saturate(vVertexColor.$VC);",
+            "    #endif",
+            "}"
+        ].join('\n');
```

Comment:
If we're making glTF 2.0 our natively supported graphics format, should we be making these chunk overrides part of the core shader chunk system?

Reply:
Yes that would be great. Will add a TODO as followup work.